csm_web/frontend/src/components/section/MentorSectionInfo.tsx

```diff
@@ -232,6 +251,58 @@ export default function MentorSectionInfo({
           </InfoCard>
         </div>
       </div>
+      <DropSection sectionId={sectionId} />
     </React.Fragment>
   );
 }
+
+interface DropSectionProps {
+  sectionId: number;
+}
+
+enum DropSectionStage {
+  INITIAL = "INITIAL",
+  CONFIRM = "CONFIRM",
+  DROPPED = "DROPPED"
+}
+
+function DropSection({ sectionId }: DropSectionProps) {
+  const sectionDropMutation = useDropSectionMutation(sectionId);
+  const [stage, setStage] = useState<DropSectionStage>(DropSectionStage.INITIAL);
+
+  const performDrop = () => {
+    sectionDropMutation.mutate(undefined, {
+      onSuccess: () => {
+        // console.log(sectionId)
+        setStage(DropSectionStage.DROPPED);
+      }
+    });
+  };
```

Comment:
There should be a failure case handled here (ex. if there are students in the section). The query hook will also need to be modified to raise this error as well.